dnd5_monsters/dnd5_monster.py

```diff
@@ -103,19 +103,11 @@ def attacks_string_to_display_string(self):
 
     def actions_dictionary_to_string(self):
         """Returns a string from dictionaries made of items with name / description ready to print"""
-        resulting_string = ""
-        for item in self.actions:
-            resulting_string += '\t' + item["name"] + "\n\t\t" + item["description"] + "\n"
-        resulting_string = resulting_string[:-1]
-        return resulting_string
+        return '\n'.join(['\t' + item["name"] + "\n\t\t" + item["description"] + "\n" for item in self.actions])
```

Comment:
I think it would be a little easier to read if it had the loop outside of the return